### DISCUSSION TITLE
How can I get the new latitude and longitude from user draggable event on react native?

### DISCUSSION THREAD

**Discussion by simancaaswork** [rnmapbox/maps#1758] [Q&A]
I have this initial location: [-69.365805, 10.062641] where I'm puttin on the marker. When the user press this marker an move it on to another position on the map, how can I get the new coordinates?

![Screen Shot 2022-03-01 at 11 36 42 PM](https://user-images.githubusercontent.com/63147124/156290262-13c7825e-56fc-409b-a961-1b0e3f6d951e.png)

**Comment by Gnative** [comment]
@simancaaswork   - I think the `MapboxGL.MarkerView `on iOS is an extension of `MapboxGL.PointAnnotation` which has `onDragStart` and `onDragEnd` functions that return coordinates.
